pkg/api/v1/interaction.proto

```
@@ -2,8 +2,28 @@ syntax = "proto3";
 
 package rig.v1;
 
+import "google/protobuf/empty.proto";
```

Buf (1.65.0)

[error] Import "google/protobuf/empty.proto" is unused.

(IMPORT_USED)